layers/+completion/helm/funcs.el

```diff
@@ -523,7 +328,11 @@ Removes the automatic guessing of the initial value based on thing at point. "
     (set-text-properties 0 (length input) nil input)
     (helm-find-files-1 input)))
 
- ;; Key bindings
+(byte-compile 'spacemacs/helm-find-files)
```

Comment:
This should be removed, otherwise you compile each time the file is loaded. Instead we should rely on the ability to auto-compile all spacemacs files, this is a setting in your dotfile.